.github/workflows/test.yml

```diff
@@ -73,6 +73,53 @@ jobs:
       - name: Run tests
         run: bundle exec rspec
 
+  postgis:
+    runs-on: ubuntu-22.04
+    strategy:
+      fail-fast: false
+      matrix:
+        rails:
+          - 7-2-stable
+          - v7.1.0
+          - v7.0.3
```

Comment:
Can you drop 7.0 and 6.1.6 from here? We ditched support for those.

Reply:
Certainly!